pom.xml

```
@@ -14,4 +14,12 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+            <version>20231013</version>
```

is there a reason you're using an old version of org.json.json?

```suggestion
            <version>20250517</version>
```

the latest version is `20250517` (https://mvnrepository.com/artifact/org.json/json)